src/Sulu/Bundle/PageBundle/Tests/Functional/Controller/NodeControllerTest.php

```diff
@@ -28,38 +29,54 @@
 class NodeControllerTest extends SuluTestCase
 {
     /**
-     * @var EntityManager
+     * @var EntityManagerInterface
      */
-    protected $em;
+    private static $em;
 
     /**
      * @var SessionInterface
      */
-    private $session;
+    private static $session;
 
     /**
      * @var SessionInterface
      */
-    private $liveSession;
+    private static $liveSession;
 
     /**
      * @var DocumentManagerInterface
      */
-    private $documentManager;
+    private static $documentManager;
+
+    /**
+     * @var PHPCRImporter
+     */
+    private static $importer;
 
     public function setUp()
     {
         parent::setUp();
 
-        $this->em = $this->getEntityManager();
-        $this->session = $this->getContainer()->get('sulu_document_manager.default_session');
-        $this->liveSession = $this->getContainer()->get('sulu_document_manager.live_session');
-        $this->documentManager = $this->getContainer()->get('sulu_document_manager.document_manager');
-
         $this->initOrm();
         $this->initPhpcr();
     }
 
+    protected static function bootKernel(array $options = [])
```

Comment:
Why is that in combination with all the static values only necessary in this test? :thinking:

Reply:
I am missing an answer here :slightly_smiling_face:

Reply:
Seems one of the only tests binding services to class variables which access the `kernel` service and createClient (which destroy and create a new kernel) is called after in its tests.